classes/classes/Output.as

```diff
@@ -60,7 +53,6 @@ package classes
 			text = kGAMECLASS.parser.recursiveParser(text);
 			record(text);
 			_currentText += text;
-			//if (debug) mainView.setOutputText(_currentText);
```

Comment:
I doubt, that this will ever return. No need to leave this here till hell freezes over. In addition we have VCS aka Git.